### PR TITLE
added content to contact and prjects sections

### PR DESCRIPTION
**Pull request by CharlieMantium**

### CODE REVIEW

File: src/components/ContactSection/ContactSection.js [hunk 1]
```diff
@@ -1,14 +1,65 @@
...
+import styled from 'styled-components';
 import { FormattedMessage } from 'gatsby-plugin-intl';
+import { LinkedinSquare } from '@styled-icons/boxicons-logos/LinkedinSquare';
+import { Envelope } from '@styled-icons/boxicons-regular/Envelope';
+import { Phone } from '@styled-icons/boxicons-regular/Phone';
 
-import SectionHeader from 'styles/elements/SectionHeader';
+import { colors, breakPoints } from 'styles/base';
 import SectionWrapper from 'styles/elements/SectionWrapper';
+import SectionHeader from 'styles/elements/SectionHeader';
```
Comment (Lukasz-Semik) [inline src/components/ContactSection/ContactSection.js]: keeping react Components in `styles` folder is very confusing. Let's thing about your architecture. Try to be generic and don't repeat yourself, but not like in rush. Think if this component will be reused (it has no so complicated styles). 

If you need some generic styled components like wrappers, maybe it is worth to create folder wrappers.

File: src/components/ContactSection/ContactSection.js [hunk 2]
```diff
@@ -1,14 +1,65 @@
...
+const ContactSectionItemWrapper = styled(SectionItemWrapper)`
+  width: 80%;
+
+  @media (min-width: ${breakPoints.desktop}) {
+    width: 40%;
+  }
+`;
+
+const LinkedInLink = styled.a`
+  text-decoration: none;
```
Comment (Lukasz-Semik) [inline src/components/ContactSection/ContactSection.js]: I would put it to global styles. Probably you want to reset those default styles.

File: src/components/ContactSection/ContactSection.js [hunk 3]
```diff
@@ -1,14 +1,65 @@
...
+
+  :visited {
```
Comment (Lukasz-Semik) [inline src/components/ContactSection/ContactSection.js]: I think that you have a typo here `&:visited`. Also I would move to global styles some basic reset of `a`, as well as `button`, etc.

File: src/components/ContactSection/ContactSection.js [hunk 4]
```diff
@@ -1,14 +1,65 @@
...
 
 const ContactSection = () => (
   <SectionWrapper id="contact">
     <SectionHeader>
       <FormattedMessage id="header.contactSection" />
     </SectionHeader>
+    <ContactSectionItemWrapper>
+      <LinkedInLink
+        href="https://www.linkedin.com/in/karol-zieniewicz"
+        target="_blank"
```
Comment (Lukasz-Semik) [inline src/components/ContactSection/ContactSection.js:32]: read about `rel` and implement it here

File: src/components/ContactSection/ContactSection.js [hunk 5]
```diff
@@ -1,14 +1,65 @@
...
+      >
+        <SectionItemIcon>
+          <LinkedinSquare />
+        </SectionItemIcon>
+      </LinkedInLink>
+      <LinkedInLink
+        href="https://www.linkedin.com/in/karol-zieniewicz"
+        target="_blank"
+      >
+        <p>Contact me on LinkedIn</p>
```
Comment (Lukasz-Semik) [inline src/components/ContactSection/ContactSection.js]: what about transations?

File: src/components/ProjectsSection/ProjectsSection.js [hunk 6]
```diff
@@ -1,14 +1,65 @@
...
+import SectionItemWrapper from 'styles/elements/SectionItemWrapper';
+import SectionItemIcon from 'styles/elements/SectionItemIcon';
+
+const ProjectsSectionItemWrapper = styled(SectionItemWrapper)`
+  flex-direction: column;
+  height: 43%;
+  width: 90%;
+
+  @media (min-width: ${breakPoints.desktop}) {
+    flex-direction: ${({ reversed }) => (reversed ? 'row-reverse' : 'row')};
```
Comment (Lukasz-Semik) [inline src/components/ProjectsSection/ProjectsSection.js]: let's keep with `is` etc prefix convention.

File: src/components/ProjectsSection/ProjectsSection.js [hunk 7]
```diff
@@ -1,14 +1,65 @@
...
+    width: 60%;
+  }
+`;
+
+const ProjectsSectionItemIcon = styled(SectionItemIcon)`
+  width: 50%;
+  max-width: 150px;
```
Comment (Lukasz-Semik) [inline src/components/ProjectsSection/ProjectsSection.js]: if you have no set 62.5% for font-sizes and you are using `px` here, I would suggest to use `rem` function from `polished` - btw, awesome package to use with styled-components

File: src/components/ProjectsSection/ProjectsSection.js [hunk 8]
```diff
@@ -1,14 +1,65 @@
...
+
+  @media (min-width: ${breakPoints.desktop}) {
+    width: 30%;
+    max-width: unset;
```
Comment (Lukasz-Semik) [inline src/components/ProjectsSection/ProjectsSection.js]: `auto` ?

File: src/pages/index.js [hunk 9]
```diff
@@ -17,6 +17,5 @@ const IndexPage = () => {
 
 export default IndexPage;
 
-// TODO: add gatsby-plugin-eslint
-// TODO: babel configuration
+// TODO: create screen shots from hangman and school-app
```
Comment (Lukasz-Semik) [inline src/pages/index.js]: maybe you can create a todo.txt file? 
todos in files could be placed in very place to be implemented.

File: src/styles/elements/SectionHeader/SectionHeader.js [hunk 10]
```diff
@@ -3,7 +3,6 @@ import styled from 'styled-components';
 import { colors, spacing, breakPoints } from 'styles/base';
 
 const SectionHeader = styled.h2`
-  margin-top: ${spacing.xxxxlSize};
```
Comment (Lukasz-Semik) [inline src/styles/elements/SectionHeader/SectionHeader.js:6]: to be hones, those spacings are quite shit. I am wondering when you come to:
`xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxSize`
Why not just use value here? It seems to be not very scalable.

File: src/components/ProjectsSection/ProjectsSection.js (hunk 8 above)
Comment (CharlieMantium) [reply src/components/ProjectsSection/ProjectsSection.js]: none :)

File: src/components/ContactSection/ContactSection.js [hunk 11]
```diff
@@ -5,11 +5,11 @@ import { LinkedinSquare } from '@styled-icons/boxicons-logos/LinkedinSquare';
 import { Envelope } from '@styled-icons/boxicons-regular/Envelope';
 import { Phone } from '@styled-icons/boxicons-regular/Phone';
 
-import { colors, breakPoints } from 'styles/base';
-import SectionWrapper from 'styles/elements/SectionWrapper';
-import SectionHeader from 'styles/elements/SectionHeader';
-import SectionItemWrapper from 'styles/elements/SectionItemWrapper';
-import SectionItemIcon from 'styles/elements/SectionItemIcon';
+import { breakPoints } from 'styles/base';
+import SectionWrapper from '../SectionWrapper';
```
Comment (Lukasz-Semik) [inline src/components/ContactSection/ContactSection.js]: Let's create styled file, export all those styled component by name. It doesn't have sense to keep separate file for each styled components

File: src/components/ContactSection/ContactSection.js [hunk 12]
```diff
@@ -19,36 +19,32 @@ const ContactSectionItemWrapper = styled(SectionItemWrapper)`
...
-`;
-
 const ContactSection = () => (
   <SectionWrapper id="contact">
     <SectionHeader>
       <FormattedMessage id="header.contactSection" />
     </SectionHeader>
     <ContactSectionItemWrapper>
-      <LinkedInLink
+      <a
```
Comment (Lukasz-Semik) [inline src/components/ContactSection/ContactSection.js:30]: why this change? have you resigned from styles?

File: src/components/LandingSection/LandingSection.js [hunk 13]
```diff
@@ -21,7 +22,7 @@ const LandingHeader = styled.h1`
 const ChangeLocaleWrapper = styled.div`
   position: fixed;
   top: ${spacing.xlSize};
-  left: -${spacing.xxxxlSize};
+  left: -100px;
```
Comment (Lukasz-Semik) [inline src/components/LandingSection/LandingSection.js:25]: maybe by next PR, I suggest to switch to rem. For example, you can use this `polished` package. You should be happy with it :P

File: src/components/ContactSection/ContactSection.js (hunk 12 above)
Comment (Lukasz-Semik) [reply src/components/ContactSection/ContactSection.js:30]: ok, nvm i see why ;)

File: src/intl/pl.json [hunk 14]
```diff
@@ -8,5 +8,14 @@
     "landingSection": "Start",
     "contactSection": "Kontakt",
     "projectsSection": "Projekty"
+  },
+  "linkTo": {
+    "linkedIn": "Najdź mnie na LinkedIn"
```
Comment (Lukasz-Semik) [inline src/intl/pl.json]: `Najdź` ? not `Znajdź`?

File: src/styles/base.js [hunk 15]
```diff
@@ -16,7 +16,6 @@ export const spacing = {
   xlSize: '20px',
   xxlSize: '35px',
   xxxlSize: '50px',
```
Comment (Lukasz-Semik) [inline src/styles/base.js:18]: I would remove all of them

**Review by Lukasz-Semik — APPROVED** [review]
